shared/config.ts

```diff
@@ -6,24 +8,19 @@
 export const isRunningLocally =
 	!process.env.LAMBDA_TASK_ROOT && !process.env.AWS_EXECUTION_ENV;
 
-export const DATABASE_NAME: string = isRunningLocally
```

Comment:
moving this to the `rds` module so that projects which don't use the db (and so don't have the relevant environmental variables) can import values from this config file